alluxio/posix/fileimpl.py

```diff
@@ -0,0 +1,219 @@
+import logging
+import os
+import re
+import threading
+import fsspec
+
+from alluxio.posix.config import ConfigManager
+from alluxio.posix.const import Constants
+from alluxio.posix.exception import UnsupportedDelegateFileSystemError
+
+local_open = os.path
+local_stat = os.stat
+local_listdir = os.listdir
+local_rename = os.rename
+local_close = os.close
+local_mkdir = os.mkdir
+local_remove = os.remove
+local_rmdir = os.rmdir
+
+
+def open(file: str, mode: str = "r", **kw):
+    logging.info("DelegateFileSystem opening file: %s", file)
+    instance = DelegateFileSystem.instance
+    fs = instance.get_file_system(file)
+    if fs:
+        try:
+            return fs.open(file, mode, **kw)
```

Comment:
Through fspec reflection, parameters can be passed directly into ufs client. Is there no need to verify the legality of config parameters?